<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>995</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>不应编码诊断</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="filterDiag"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="ExceptDiag"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>不应编码手术</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="filterOper"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="ExceptOper"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
